scripts/format.sh

```diff
@@ -22,5 +22,5 @@ yarn --cwd "frontend" pretty-quick --staged
 # "--diff-filter=ACMR" only lists files that are [A]dded, [C]opied, [M]odified,
 # or [R]enamed; we don't want to try to format files that have been deleted.
 if command -v "black" > /dev/null; then
-  git diff --diff-filter=ACMR --name-only --cached | grep -E "\.pyi?$" | xargs black
+  git diff --diff-filter=ACMR --name-only --cached | grep -E "\.pyi?$" | grep -v "hello.py" | xargs black
```

Comment:
Not sure I like this that much. @kantuni is there a better way to exclude a file?

Reply:
I believe you can use `# fmt: off` at the start of some lines of code you want to ignore and `# fmt: on` at the end.

https://github.com/psf/black#the-black-code-style